modules/geo-layers/src/mvt-layer/mvt-layer.ts

```diff
@@ -22,28 +38,74 @@ const defaultProps = {
   binary: true
 };
 
-export default class MVTLayer extends TileLayer {
-  initializeState() {
+export type TileJson = {
+  tilejson: string;
+  tiles: string[];
+  // eslint-disable-next-line camelcase
+  vector_layers: any[];
+  attribution?: string;
+  scheme?: string;
+  maxzoom?: number;
+  minzoom?: number;
+  version?: string;
+};
+
+export type MVTLayerProps = {
```

Comment:
- Missing GeoJsonLayer prop types
- User-facing `*LayerProps` types should merge all its ancestor prop types (TileLayerProps, LayerProps)

Reply:
How about typing `GeoJsonLayer` props in separate PR? It's quite big ...